google/cloud/spanner/samples/samples.cc

```diff
@@ -1363,9 +1357,6 @@ void ReadWriteTransaction(google::cloud::spanner::Client client) {
     // Commit() only retries on StatusCode::kAborted.
     if (!row) return std::move(row).status();
     return std::get<0>(*std::move(row));
-    // Throw an exception because this should terminate the transaction.
-    throw std::runtime_error("Key not found (" + std::to_string(singer_id) +
```

Comment:
Sync up. I noticed and removed this one earlier today.

Reply:
Yikes.  That is all I was really trying to do with this whole `throw` thread (because I wanted to use this code as part of a snippet).